packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@heetch/flamingo-react",
-  "version": "1.15.2",
```

Comment:
Shouldn't this be done after merging, and by Lerna automatically?

Reply:
Yes @obedparla, we must not update manually any version 🙏

Reply:
Yes you are right, thanks